src/cli.ts

```diff
@@ -113,6 +118,8 @@ async function calc(
     result = await calculateGapsInCare(measureBundle, patientBundles, calcOptions, valueSetCache);
   } else if (program.outputType === 'dataRequirements') {
     result = calculateDataRequirements(measureBundle, calcOptions);
+  } else if (program.outputType === 'libraryDataRequirements') {
+    result = calculateLibraryDataRequirements(measureBundle, calcOptions);
```

Comment:
This is a little misleading since library data requirements does not make use of a measure. Would it be too much to rename `measureBundle` to `measureOrLibraryBundle`? I can't think of a better name for this lol. At the very least, we should add a comment about the bundle not containing a measure

Reply:
Yeah I agree! I didn't want to make another input since it would've been a bit unnecessarily complicated but I can always do that instead.